src/utils/request-queue.js

```diff
@@ -33,6 +34,10 @@ class RequestQueue {
     if (this.sendTrackingEvents && !this.humanity.isBot()) {
       const queue = RequestQueue.get();
 
+      if (requestContainsPii(url, body)) {
+        return;
+      }
```

Comment:
We could probably move this right before line 35 where we get the queue.

Reply:
🧠

Reply:
👍